apps/arena/lib/arena/game/crate.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Arena.Game.Crate do
+  @moduledoc """
+  Module to handle crate logic
+  """
+  def take_damage(crate, damage) do
+    update_in(crate, [:aditional_info, :health], fn current_health -> max(current_health - damage, 0) end)
+  end
+
+  def alive?(crate) do
+    crate.aditional_info.health > 0
+  end
+
+  def interactable_crates(crates) do
+    Map.filter(crates, fn {_crate_id, crate} -> crate.aditional_info.status != :DESTROYED end)
```

Comment:
Any reason for this to not be based on `alive?/1` ?

Reply:
no reason at all, changed!